docs/examples/example_merge_species.py

```diff
@@ -0,0 +1,77 @@
+from pyrokinetics import Pyro, template_dir
+from pint import UnitRegistry
+import numpy as np
+
+# data path
+spr_path = '/home/ab1209/pyrokinetics/SPR-JETTO/qmin_264_imp'
+eqb_file = spr_path + '/jetto.eqdsk_out'
+kin_file = spr_path + '/jetto.jsp'
+psi_n = 0.5
+
+
+# load pyro object
+pyro = Pyro(
+        eq_file=eqb_file,
+        kinetics_file=kin_file,
+        )
+kin = pyro.kinetics
+eqb = pyro.eq
+ureg = UnitRegistry()
+
+
+# inform species
+for count,species in enumerate(kin.species_data.keys()):
+
+    charge = kin.species_data[f'{species}'].charge(psi_n*ureg.dimensionless)
+    print(f'{species}, Z: {charge.magnitude}')
+
+
+# check with users species to merge
+spec_list = []
+nspec = int(input("Enter number of species to merge : "))
```

Comment:
Fine for an example, but we might want this functionality to be scriptable -- move the functionality into a function which takes all required input as an argument. That way example scripts like this can still read user input (although perhaps better using an argument parser than relying on input interaction) but the code can easily be reused in automated workflows.

Reply:
Adding a method to the kinetic species object seems like the cleanest way to do this so its built into pyro